options.yml

```diff
@@ -1,66 +1,217 @@
-window:
-  type:
-    type: choice
-    choices: [window, borderlessWindow, borderlessFullscreen]
-    default: window
-  width:
-    type: integer
-    range:
-    min: 0
-    max: 20000
-    default: 1280
-  height:
-    type: integer
-    range:
-    min: 0
-    max: 20000
-    default: 720
-  pos:
-    type: choice
-    choices: [middle, topLeft, bottomLeft, topRight, bottomRight]
-    default: middle
-  continueOutOfFocus:
-    type: choice
-    choices: [pause, continue, render]
-    default: pause
-graphic:
-  api:
-    type: choice
-    choices: [DirectX, OpenGL]
-    default: DirectX
-  filterLinear:
-    type: boolean
-    default: true
-  vsync:
-    type: boolean
-    default: true
-  waitWithGLFinish:
-    type: boolean
-    default: false
-  pixFormat:
-    type: choice
-    choices: [argb1555, rgb565]
-    default: argb1555
-  debug:
-    type: choice
-    choices: [none, enabled, debugContextEnabled]
-    default: none
-control:
-  clipCursor:
-    type: boolean
-    default: true
-  scrollActive:
-    type: boolean
-    default: true
-  margin:
-    type: integer
-    range:
-      min: 0
-      max: 1000
-    default: 0
-  padding:
-    type: integer
-    range:
-      min: 0
-      max: 1000
-    default: 0
\ No newline at end of file
+- display: GroupBox
+  header: '{{general}}'
+  text: '{{general_descr}}'
+  hasHeader: true
+  text: Replace the Graphics API. Allows for recording streams with software such as OBS.
+  columns: 1
```

Comment:
It is okay to have this, but I am likely going to deprecate it soon :)
So you could remove it already.

Reply:
Done